client-js/connection.js

```diff
@@ -42,7 +42,7 @@ module.exports = function () {
         renderTesting();
         $.ajax({
             type: "POST",
-            url: "/connections/test",
+            url: baseUrl + "/sqlpad/connections/test",
```

Comment:
Thanks, fixed! Once we're comfortable with OAuth authentication, this should be good to go.